contrib/node/src/python/pants/contrib/node/tasks/node_resolve.py

```diff
@@ -86,6 +83,18 @@ def prepare(cls, options, round_manager):
     for resolver in cls._resolver_by_type.values():
       resolver.prepare(options, round_manager)
 
+  @classmethod
+  def register_options(cls, register):
+    super(NodeResolve, cls).register_options(register)
+    register('--install-invalidating-files', type=list,
```

Comment:
This should be marked `advanced=True` probably.

Reply:
Solved in the last commit :)